### PR TITLE
VACMS-9016: Updates Cypress test for content release.

### LINKED ISSUE #9016: Fix cypress test for testing frontend build trigger
## Description

The content release Cypress test https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/tests/cypress/integration/behavioral/content_release.feature isn't working in the feature branch for 8108. It should work.

Ref #8108

## Acceptance Criteria
- [ ] Content release Cypress test works as expected for https://github.com/department-of-veterans-affairs/va.gov-cms/pull/8934

### CMS Team
Please check the team(s) that will do this work.

- [x] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

Closes #9016.

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Do this
   - [ ] Validate that
2. Then
   - [ ] Validate that
3. Then validate Acceptance Criteria from issue
   - [ ] a
   - [ ] b
   - [ ] c

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_consumers/src/Git/GitFactory.php [hunk 1]
```diff
@@ -35,9 +61,7 @@ public function get(string $repositoryRoot) : GitInterface {
    *   The Git object.
    */
   public function getWebRepository() : GitInterface {
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_consumers/src/Git/GitFactory.php:63]: Is it me or is it hard to say this without sounding like Bugs Bunny?

Comment (ndouglas) [reply docroot/modules/custom/va_gov_consumers/src/Git/GitFactory.php:63]: 😂

**Comment by ndouglas** [comment]
Finally getting some uninterrupted time to work on this today 😩 

If I dispatch a build locally from a user with the `content_admin` role, I see the following:

Screen Shot 2022-06-07 at 9 27 33 AM

Poking around a bit, it appears to be because the `advancedqueue` queue is not processed automatically by `ddev`:

Screen Shot 2022-06-07 at 9 31 53 AM

Running it:

Screen Shot 2022-06-07 at 9 32 52 AM

That's fine and dandy.  But then, returning to `admin/content/deploy`:

Screen Shot 2022-06-07 at 9 33 21 AM

And there it will remain; the state never gets reset.

**Comment by cweagans** [comment]
> And there it will remain; the state never gets reset.

So at the point where a build has been dispatched, site relies on being informed by the content release that something is happening (like this: https://github.com/department-of-veterans-affairs/content-build/blob/main/.github/workflows/content-release.yml#L109-L115). I think there's also a drush command to advance the state if you'd prefer.

**Comment by cweagans** [comment]
(also, feel free to push this stuff directly into the 8108 branch -- it might be easier to stay in sync that way)

**Comment by ndouglas** [comment]
> So at the point where a build has been dispatched, site relies on being informed by the content release that something is happening (like this: https://github.com/department-of-veterans-affairs/content-build/blob/main/.github/workflows/content-release.yml#L109-L115). I think there's also a drush command to advance the state if you'd prefer.

But a build _hasn't_ been dispatched, according to the message.  That's not a big deal; I was more just explaining why I took the approach I did with the Cypress tests.

>(also, feel free to push this stuff directly into the 8108 branch -- it might be easier to stay in sync that way)

Will do!  I just didn't wanna come into your PR and change things willy-nilly 🙂

**Pull request merged**